src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlRedirection/XamlMember.cs

```diff
@@ -49,5 +50,20 @@ public bool IsAttachable
 			=> _declaringType != null ? _isAttachable : XamlConfig.IsUnoXaml ? _unoMember.IsAttachable : _msMember.IsAttachable;
 
 		public override string ToString() => XamlConfig.IsUnoXaml ? _unoMember.ToString() : _msMember.ToString();
+
+		public bool Equals(XamlMember other)
+			=> _name.HasValue()
+			? (_name == other._name && _declaringType == other._declaringType && _isAttachable == other.IsAttachable)
```

Comment:
You should check if `other` is 'null'

Reply:
I surely home to remove this code once we can integrate the proper System.Xaml :)